php/Terminus/Auth.php

```diff
@@ -36,12 +36,58 @@ public static function loggedIn() {
   }
 
   /**
+   * Execute the login based on a new refresh token
+   *
+   * @param [string] $token Refresh token to initiate login with
+   * @return [boolean] True if login succeeded
+   */
+  public function logInViaRefreshToken($token) {
+    $options = array(
+      'headers' => array('Content-type' => 'application/json'),
+      'cookies' => array('Authorization: Bearer' => $token),
+    );
+    $logger_context = compact('token');
+
+    $this->logger->info(
+      'Logging in via refresh token {token}',
+      $logger_context
+    );
+    $response = TerminusCommand::request(
+      'auth/refresh',
+      '',
+      '',
+      'POST',
+      $options
+    );
+    if ($response['status_code'] != '200') {
+      throw new TerminusException(
+        'Login via refresh token {token} was unsuccessful.',
+        $logger_context,
+        1
+      );
+    }
+
+    $this->setInstanceData(
+      array(
+        'user_uuid'           => $response['data']->id,
+        'session'             => $response['data']->session,
```

Comment:
It will! We haven't built that yet. I'll try and get a PR with that functionality in place up soon. This is pretty much exactly right except I think the refresh token should be posted in the body.

Eventually we will have this function return a JWT instead of a session identifier so we may want to change the 'session' key in the output to 'id_token' so that it remains correct in the future.